diff --git a/src/ui/index.tis b/src/ui/index.tis
index 1dff3cecb..7684d3efa 100644
--- a/src/ui/index.tis
+++ b/src/ui/index.tis
@@ -45,8 +45,7 @@ class ConnectStatus: Reactor.Component {
         } else if (connect_status == 0) {
             return translate('connecting_status');
         }
-        if (!handler.using_public_server()) return translate('Ready');
-        return <span>{translate("Ready")}, <span .link #setup-server>{translate("setup_server_tip")}</span></span>;
+        return translate('Ready');
     }
 
     event click $(#start-service) () {
